src/index.js

```diff
@@ -1,3 +1,26 @@
-const inputFile = require("./input.json");
+const inputFile = require("../input/input"); // assumes an array of JSON objects
+const { updateApplicationStatus, validate } = require("./utils/utils");
 
+const errors = [];
+data = inputFile.map(app => {
+    try {
+        if(validate(app)){
+            return app;
+        }
+        
+    } catch (error) {
+        errors.push(error);
+        return;
```

Comment:
I think .map() will require a return of some data even if is an error.

Reply:
thanks for the info, I'll implement that differently

Reply:
you were definitely right, it returns undefined to the new array with the map method